syntax = "proto3";

package mapreduce_common;


message EmptyMessage {

}